apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/SpannerGranularAccessTest.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Spanner.Data.IntegrationTests;
+
+[Collection(nameof(SpannerDatabaseRoleFixture))]
+[CommonTestDiagnostics]
+public class SpannerGranularAccessTest
+{
+    private readonly SpannerDatabaseRoleFixture _fixture;
+
+    public SpannerGranularAccessTest(SpannerDatabaseRoleFixture fixture) =>
+        _fixture = fixture;
+
+    [SkippableFact]
+    public async Task ReadWriteWithDbRoleTest()
```

Comment:
Please split this test in two, one for testing that read is successfull and one for testing that write is not.